include/deal.II/matrix_free/operators.h

```diff
@@ -665,6 +667,24 @@ namespace MatrixFreeOperators
     apply(const VectorizedArrayType *in_array,
           VectorizedArrayType *      out_array) const;
 
+    /**
+     * This operation applies the inverse @ref GlossMassMatrix "mass matrix"
+     * operation on an input array with local dyadic-valued coefficients.
+     * The second-rank tensor at each quadrature point defines a linear operator
+     * on a vector holding the dof components. It is assumed that the passed
+     * input and output arrays are of correct size, namely
+     * FEEvaluation::dofs_per_cell long. The `in_array` and `out_array`
```

Comment:
```suggestion
     * FEEvaluation::dofs_per_cell long. The @p in_array and @p out_array
```

Reply:
Should I keep it? I think the ticks are easier to read in the source code.

Reply:
Personally, I prefer to use `@p` (for the reason you have provided); but I let you decide. I am fine with any version.